src/components/doc-explorer/Argument.css

```diff
@@ -65,11 +65,4 @@
   & .arg-description > p {
     margin: 0;
   }
-
-  & .arg-description:before {
-    display: block;
-    content: "#";
-    float: left;
-    margin-right: var(--spacing-unit);
-  }
```

Comment:
Can you please explain why you removed it?

Reply:
I've removed it for consistency with other descriptions (type descriptions and field descriptions have no leading `#`). I've spent some time to figure out why this symbol appears (I thought that markdown is broken).